src/libraries/Microsoft.PowerFx.Repl/Repl.cs

```diff
@@ -37,6 +37,13 @@ public class PowerFxREPL
         // Allow repl to create new UserDefinedFunctions. 
         public bool AllowUserDefinedFunctions { get; set; }
 
+        /// <summary>
+        /// Enable the Import() function for importing modules. 
+        /// Defaults to false. 
+        /// </summary>
+        [Obsolete("preview")]
+        public bool AllowImport { get; set; }
```

Comment:
PAC CLI is only place that includes repl. This flag is false by default, so no behavior change.